doc/source/services.rst

```diff
@@ -52,11 +52,20 @@ more flexible, supporting multiple services and a wider range of
 options.
 
 To create the service, create a python script with your service code
-and add a :code:`--service=myservice:/path/to/myservice.py` argument
+and add a :code:`--service=myservice:myservice.py` argument
```

Comment:
Why the removal of `/path/to/` ?

Reply:
Yeah, you're right, and the absolute path was probably a typo of the previous contributor.

But, I think that is nice to explain that `myservice.py` is actually the entrypoint of the service, and could be placed into a subfolder?

How it sounds `--service=myservice:PATH_TO_SERVICE_PY`?

( We can also add a note which explains that `PATH_TO_SERVICE_PY` could be a relative path to the service entrypoint like `services/myservice.py`. )